src/pages/About.jsx

```diff
@@ -1,7 +1,113 @@
 import React from 'react';
+import TeamMemberCard from '../components/TeamMemberCard';
+import avatar from '../assets/images/square.png';
 
 const About = () => {
-  return <></>;
+  const frontend = [
+    {
+      image: avatar,
+      name: 'Evgenii',
+      role: 'Frontend',
+    },
+    {
+      image: avatar,
+      name: 'Brandon',
+      role: 'Frontend',
+    },
+    {
+      image: avatar,
+      name: 'Liuba',
+      role: 'Frontend',
+    },
+  ];
+
+  const frontendCards = frontend.map((item, index) => (
+    <TeamMemberCard
+      key={index}
+      img={item.image}
+      name={item.name}
+      role={item.role}
+    />
+  ));
+
+  const backend = [
+    {
+      image: avatar,
+      name: 'Valentina',
+      role: 'Backend',
+    },
+    {
+      image: avatar,
+      name: 'Tetiana',
+      role: 'Backend',
+    },
+  ];
+
+  const backendCards = backend.map((item, index) => (
+    <TeamMemberCard
+      key={index}
+      img={item.image}
+      name={item.name}
+      role={item.role}
+    />
+  ));
+
+  const mentors = [
+    {
+      image: avatar,
+      name: 'Sergey',
+      role: 'Mentor',
+    },
+    {
+      image: avatar,
+      name: 'Dan',
+      role: 'Mentor',
+    },
+    {
+      image: avatar,
+      name: 'Vadim',
+      role: 'Mentor',
+    },
+  ];
+
+  const mentorCards = mentors.map((item, index) => (
+    <TeamMemberCard
+      key={index}
+      img={item.image}
+      name={item.name}
+      role={item.role}
+    />
+  ));
+
+  return (
+    <div className="w-full flex flex-col items-center bg-white">
+      <h2 className="font-headings font-bold text-3xl md:text-4xl my-7 md:my-10">
+        Meet our team
+      </h2>
+      <div className="flex flex-col md:flex-row justify-between gap-2 md:gap-16">
+        {frontendCards}
+      </div>
+      <div className="flex flex-col md:flex-row justify-around gap-2 md:gap-32">
+        {backendCards}
+      </div>
+      <div className="flex flex-col md:flex-row justify-between gap-2 md:gap-16">
+        {mentorCards}
+      </div>
+      <h2 className="font-headings font-bold text-3xl md:text-4xl my-7 md:my-10">
+        About Re:Books
+      </h2>
+      <p className="px-10 pb-10 flex flex-col items-center w-full font-body text-justify">
```

Comment:
Minor Tailwind suggestions:
- Horizontal padding should be 60 pixels on the desktop version
- `flex w-full flex-col items-center` can be removed
- Font size should be 24px and line height should be 52px